en/mkdocs.yml

```diff
@@ -42,11 +42,26 @@ theme:
   feature:
     tabs: true
   language: 'en'
+  
+# Navigation
+nav:
+- Basic Tutorials:
+      - 'Adaptive Authentication': 
+        - 'Adaptive Authentication Overview': 'Tutorials/adaptive-authentication-overview.md'
```

Comment:
Shall we make the folder names simple?

Reply:
+1. Fixed with 58aa2d6f6b18b5344a19c4fc88cb25f1f30756e8